packages/datetime/src/datePickerCore.tsx

```diff
@@ -13,7 +13,8 @@ export interface IDatePickerLocaleUtils {
     formatWeekdayShort: (weekday: number, locale: string) => string;
     formatWeekdayLong: (weekday: number, locale: string) => string;
     getFirstDayOfWeek: (locale: string) => number;
-    getMonths: (locale: string) => string[];
+    // tslint:disable-next-line:max-line-length
+    getMonths: (locale: string) => [string, string, string, string, string, string, string, string, string, string, string, string];
```

Comment:
nit: Would prefer to give this type a clearer name. When I first saw this, I had to count the number of `string`s before I understood what this was.